frontend/src/components/RepositoryForm.vue

```diff
@@ -127,7 +127,8 @@
         }}
       </div>
     </div>
-    <div>
+    <!-- TODO(Steven): remove the isDev flag when release v1.1.0 -->
+    <div v-if="isDev()">
```

Comment:
I wonder `isDev` is a boolean value or a function?

Reply:
It'a function https://github.com/bytebase/bytebase/blob/main/frontend/src/utils/util.ts#L12